### PR TITLE
Ch redesign about page 126584711

### PR DESCRIPTION
**Pull request by andela-cnwosu**
#[126584711] About Page Redesign

#### What does this PR do?
Redesign of About page and fixing of erratic test bugs

#### Description of Task to be completed?
The redesign is necessary to make the page responsive and to allow for dynamic generation of statistics about the site. Erratic feature test bugs were also fixed.

#### How should this be manually tested?
Clone repo on https://github.com/andela/eventx, start the server and visit page at localhost:3000

#### Screenshots
screencapture-localhost-3000-about-1472045909288
screencapture-localhost-3000-about-1472045873489

### CODE REVIEW

File: app/presenters/welcome/welcome_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module Welcome
+  class WelcomePresenter
+    def attendee_count
+      Attendee.count
```
Comment (andela-sadefila) [inline app/presenters/welcome/welcome_presenter.rb]: Are we getting a count of ALL the attendees in our db? or attendees for some SPECIFIC events?

File: spec/features/request_refund_spec.rb [hunk 2]
```diff
@@ -22,7 +22,8 @@
     expect(page).to have_content "The event was cancelled"
     expect(page).to have_content "SUBMIT REQUEST"
     within("#refund-form") do
-      find("a", text: "SUBMIT REQUEST").click
+#      find("a", text: "SUBMIT REQUEST").click
```
Comment (andela-sadefila) [inline spec/features/request_refund_spec.rb]: NITPICK: remove unneeded commented out code.

File: app/assets/stylesheets/welcome.scss.erb [hunk 3]
```diff
@@ -114,12 +118,31 @@ html, body{
...
+}
+
+.yellow_text {
+  color: #F7B733;
+}
+
+.blue_text{
+  color: #2196F3;
+}
+
```
Comment (andela-ajamiu) [inline app/assets/stylesheets/welcome.scss.erb]: Awesome - Consistent CSS line spacing.

File: app/assets/stylesheets/welcome.scss.erb [hunk 4]
```diff
@@ -354,3 +393,137 @@ nav.custom_nav{
 a.remove-text {
   color: #ffffff !important;
 }
+
+.about-top {
+  position: relative;
+  background-image: url("https://pixabay.com/static/uploads/photo/2014/09/07/17/41/girl-438129_960_720.jpg");
```
Comment (andela-megwali) [inline app/assets/stylesheets/welcome.scss.erb]: It's not a good idea to hotlink images from a site you don't control. Better to download it and link it locally. Also for SEO purposes, the name of the image should be changed to tell us something about EventX and NOT pixabay.

File: app/assets/stylesheets/welcome.scss.erb [hunk 5]
```diff
@@ -354,3 +394,136 @@ nav.custom_nav{
...
+.about-content {
+  padding-bottom: 70px;
+  margin: auto;
+  font-size: 1.5em;
+  font-weight: 200;
+}
+
+.about-content h4 {
+  padding: 10px;
+  padding-left: 0
```
Comment (andela-cnwosu) [inline app/assets/stylesheets/welcome.scss.erb]: Put semi-colons at the end of each attribute.

File: app/assets/stylesheets/welcome.scss.erb [hunk 6]
```diff
@@ -114,12 +118,44 @@ html, body{
...
+
+.teal_text{
+  color: #4ABDAC;
+}
+
+.yellow_bg {
+  background: #F7B733;
+}
+
+.orange_bg {
```
Comment (andela-cnwosu) [inline app/assets/stylesheets/welcome.scss.erb]: Make singular rules a one-liner statement

File: app/assets/stylesheets/welcome.scss.erb [hunk 7]
```diff
@@ -354,3 +394,136 @@ nav.custom_nav{
...
+}
+
+.card.feature-card {
+  margin-left: -12px;
+  margin-right: -10px;
+}
+
+#faq-row {
+  background: url(/assets/nearfooter.png);
+  padding: 20px 0 20px;
```
Comment (andela-cnwosu) [inline app/assets/stylesheets/welcome.scss.erb]: Merge the top and bottom padding values to 20px 0

File: app/assets/stylesheets/welcome.scss.erb [hunk 8]
```diff
@@ -354,3 +394,136 @@ nav.custom_nav{
...
+  margin: 9px 0 0px;
```
Comment (andela-cnwosu) [inline app/assets/stylesheets/welcome.scss.erb]: Set margin top only

File: app/presenters/welcome/welcome_presenter.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+module Welcome
+  class WelcomePresenter
+    def user_count
```
Comment (andela-cnwosu) [inline app/presenters/welcome/welcome_presenter.rb]: change method name to more explicit names

File: app/controllers/welcome_controller.rb [hunk 10]
```diff
@@ -15,6 +15,10 @@ def popular
     renders(@events)
   end
 
+  def about
+    @presenter = Welcome::WelcomePresenter.new
```
Comment (andela-cnwosu) [inline app/controllers/welcome_controller.rb]: change variable name, presenter

File: app/views/welcome/about.html.erb [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    </div>
+  </div>
+</div>
+
+<div class="about-stats-div">
+  <div class="about">
+    <div class="about-stats row group">
+      <div class="col s12 m12 l4 padding-side-25">
+        <div class="milestones teal_bg">
+          <div class="stats-large"><%= @presenter.event_count %> </div>
```
Comment (andela-cnwosu) [inline app/views/welcome/about.html.erb]: put ruby code on next line

**Comment by andela-kadeniyi** [comment]
@andela-cnwosu Please fix your merge conflict and reach out when you are done.

**Pull request closed without merging**